README.md

```diff
@@ -71,7 +71,8 @@ matching route is found in the proxy table:
     -V, --version                    output the version number
     --ip <ip-address>                Public-facing IP of the proxy
     --port <n> (defaults to 8000)    Public-facing port of the proxy
-    
+    --storage-command <script>       The optional external executable to use for persistence
```

Comment:
Scratch my earlier suggestion of `persistent-storage`. `storage-command` still feels a bit vague to me as a name perhaps:
`--external-storage <path>   Path to executable script that stores routing table externally instead of in-memory`